src/main/java/seedu/mtracker/console/InputParser.java

```diff
@@ -171,7 +171,7 @@ public Command filterByCommandType(String[] commandComponents, ArrayList<Instrum
     }
 
     public String[] getCommandComponents(String commandInput) {
-        return commandInput.trim().split(SEPARATOR);
+        return commandInput.trim().toLowerCase().split(SEPARATOR);

```

Comment:
does this make for example my name input all lowercase? cause the name inputs might have to be uppercase

Reply:
nope, the name, price, remarks those dont use the getCommandComponents method

Reply:
oh ok nice will merge